src/field/pokemon.ts

```diff
@@ -2075,8 +2075,8 @@ export default abstract class Pokemon extends Phaser.GameObjects.Container {
           stabMultiplier.value = Math.min(stabMultiplier.value + 0.5, 2.25);
         }
 
-        const targetCount = getMoveTargets(source, move.id).targets.length;
-        const targetMultiplier = targetCount > 1 ? 0.75 : 1; // 25% damage debuff on multi-target hits (even if it's immune)
+        // 25% damage debuff on moves hitting more than one non-fainted target (regardless of immunities)
+        const targetMultiplier = targets > 1 ? 0.75 : 1;
```

Comment:
```suggestion
        const { targets, isMultiTarget } = getMoveTargets(source, move.id);
        const targetMultiplier = (isMultiTarget && targets.length > 1) ? 0.75 : 1;
```
The output of `getMoveTargets` is an object `{targets, multiple}` that gives you the battler indexes the move can legally target, as well as whether or not the move is multi-targeted. I don't think the use of `getMoveTargets` here was totally incorrect; it was just missing a check for the `multiple` output.

Reply:
Ah, I guess I misunderstood exactly what that function was doing. That does seem to work. 👍